packages/components/src/text-input/tests/chromatic/TextInput.chroma.jsx

```diff
@@ -0,0 +1,160 @@
+import { Inline, Stack } from "@components/layout";
+
+import { CrossButton } from "@components/button";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong